### PR TITLE
Add Django admin

### PR DESCRIPTION
**Pull request by alikins**
Setup django admin, and add admin setup for galaxy_api.auth

### CODE REVIEW

**Comment by cutwater** [comment]
Why don't you use standard Django user admin config?

`django.contrib.auth.admin.UserAdmin`

**Comment by alikins** [comment]
@cutwater "Why don't you use standard Django user admin config?"

Didn't know about it. updating now.

File: galaxy_api/auth/admin.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from django.contrib import admin
+from galaxy_api.auth import models
+
+
+@admin.register(models.User)
+class UserAdmin(admin.ModelAdmin):
+    list_display = ('__str__',
```
[the remark this answers is not in the document]
Comment (alikins) [reply galaxy_api/auth/admin.py]: got a name for that style so I can add it to my editor configs?

[the remark this answers is not in the document]
Comment (alikins) [reply galaxy_api/auth/admin.py]: Replaced with direct subclass of  django.contrib.auth.admin.UserAdmin.

**Pull request merged**